views/home.erb

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Donate - Survival International</title>
+    <meta name="description" content="Survival helps tribal peoples defend their lives, protect their lands and determine their own futures.">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="manifest" href="site.webmanifest">
+    <link rel="apple-touch-icon" href="icon.png">
+    <link rel="icon" href="favicon.ico">
+
+    <link rel="stylesheet" href="stylesheets/normalize.css">
+    <link rel="stylesheet" href="stylesheets/main.css">
+  </head>
+  <body>
+    <!--[if lte IE 9]>
+        <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+    <![endif]-->
+
+    <div class="wrapper">
+      <h1>Your donation</h1>
+      <form action="/donations" method="post">
```

Comment:
Am I missing something, where is the URL `/donations` defined?

Reply:
This PR is just for adding the form. The `/donations` route will be implemented in future PRs.